pkg/integrations/acs.go

```diff
@@ -27,14 +27,22 @@ type ACSIntegration struct {
 }
 
 // PersistentFlags sets the persistent flags for the ACS integration.
-func (a *ACSIntegration) PersistentFlags(p *pflag.FlagSet) {
+func (a *ACSIntegration) PersistentFlags(c *cobra.Command) {
+	p := c.PersistentFlags()
+
 	p.BoolVar(&a.force, "force", a.force,
 		"Overwrite the existing secret")
 
 	p.StringVar(&a.endpoint, "endpoint", a.endpoint,
 		"ACS service endpoint, formatted as 'hostname:port'")
 	p.StringVar(&a.token, "token", a.token,
 		"ACS API token")
+
+	for _, f := range []string{"endpoint", "token"} {
+		if err := c.MarkPersistentFlagRequired(f); err != nil {
```

Comment:
You need to clean-up the `Validate()` method, as this makes some checks redundant.

This comment is likely applicable to all the changes in `integrations`.